app.rb

```diff
@@ -4,13 +4,37 @@
 require 'byebug'
 
 require_relative 'publisher.rb'
+require_relative 'subscriber.rb'
+
+class App
+
+  attr_reader :sqs
+
+  def initialize
+    creds = Aws::Credentials.new(AWS["access_key_id"], AWS["secret_access_key"])
+    @sqs = Aws::SQS::Client.new(region: AWS["region"], credentials: creds)
+  end
+
+  def run_service(options)
+    service = options.delete_at(0)
+    case service
+    when 'publish'
+      publisher = Publisher.new(sqs)
```

Comment:
Added validations in `initializer` of objects. Since currently we are using only sleep_time validations, and defaulting them to 5, not throwing any exceptions or errors as of now.